### PR TITLE
(DOCSP-16607) Backfills Atlas CLI content for several clusters commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills content for several Atlas CLI clusters commands

_Jira tickets:_

https://jira.mongodb.org/browse/DOCSP-16607
https://jira.mongodb.org/browse/DOCSP-16610
https://jira.mongodb.org/browse/DOCSP-16611
https://jira.mongodb.org/browse/DOCSP-16612
https://jira.mongodb.org/browse/DOCSP-16614
https://jira.mongodb.org/browse/DOCSP-16616

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/atlascli/command/atlas-clusters-delete.txt [hunk 1]
```diff
@@ -14,6 +14,12 @@ atlas clusters delete
 
 Delete a cluster from your project.
 
+The command prompts 
+		you to confirm the operation when you run the command 
+		without the --force option.
```
Comment (andreaangiolillo) [inline docs/atlascli/command/atlas-clusters-delete.txt:19]: This format does not look right

File: docs/atlascli/command/atlas-clusters-onlineArchives-create.txt [hunk 2]
```diff
@@ -12,7 +12,11 @@ atlas clusters onlineArchives create
    :depth: 1
    :class: singlecol
 
-Create an online archive for a cluster.
+Create an online archive for a collection in the cluster you specify.
+
+You can create an online archive for an M10 or larger cluster.
+		
+		To learn more about online archives, see https://www.mongodb.com/docs/atlas/online-archive/manage-online-archive/.
```
Comment (andreaangiolillo) [inline docs/atlascli/command/atlas-clusters-onlineArchives-create.txt:19]: format does not seem correct

File: docs/mongocli/command/mongocli-atlas-clusters-onlineArchives-create.txt [hunk 3]
```diff
@@ -12,7 +12,11 @@ mongocli atlas clusters onlineArchives create
    :depth: 1
    :class: singlecol
 
-Create an online archive for a cluster.
+Create an online archive for a collection in the cluster you specify.
+
+You can create an online archive for an M10 or larger cluster.
+		
+		To learn more about online archives, see https://www.mongodb.com/docs/atlas/online-archive/manage-online-archive/.
```
Comment (andreaangiolillo) [inline docs/mongocli/command/mongocli-atlas-clusters-onlineArchives-create.txt:19]: 👀

File: internal/cli/atlas/clusters/onlinearchive/create.go [hunk 4]
```diff
@@ -98,8 +98,16 @@ func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{}
 	cmd := &cobra.Command{
 		Use:   "create",
-		Short: "Create an online archive for a cluster.",
-		Args:  require.NoArgs,
+		Short: "Create an online archive for a collection in the cluster you specify.",
+		Long: `You can create an online archive for an M10 or larger cluster.
+		
+		To learn more about online archives, see https://www.mongodb.com/docs/atlas/online-archive/manage-online-archive/.`,
```
Comment (andreaangiolillo) [inline internal/cli/atlas/clusters/onlinearchive/create.go:104]:
```suggestion
To learn more about online archives, see https://www.mongodb.com/docs/atlas/online-archive/manage-online-archive/.`,
```

**Review by andreaangiolillo — APPROVED** [review]
LGTM. Thanks for all the changes

**Review by zach-carr — APPROVED** [review]
LGTM

**Comment by andreaangiolillo** [comment]
yes, failures are not related. Thanks!

**Pull request merged**